modules/debezium_connection/main.tf

```diff
@@ -1,52 +1,56 @@
 locals {
+  heartbeat_insertion = <<EOF
+    INSERT INTO public.${var.events_table}
+      (id, public_subscription_id, partition_key, topic_name, tenant_id, event_json, created_at)
```

Comment:
`partition_key` (??) should probably be random as well.

Reply:
Update these to the latest column names. They should match these: https://github.com/nulogy/message-bus/blob/master/gems/nulogy_message_bus_producer/spec/dummy/db/schema.rb#L18

```
  create_table "message_bus_subscription_events", id: :uuid, default: nil, force: :cascade do |t|
    t.uuid "subscription_id", null: false
    t.string "partition_key", null: false
    t.string "topic_name", null: false
    t.uuid "company_uuid", null: false
    t.json "event_json", null: false
    t.datetime "created_at"
    t.index ["created_at"], name: "index_nulogy_mb_producer_subscription_events_on_created_at"
  end
```